Resources/Prototypes/Entities/Objects/Devices/encryption_keys.yml

```diff
@@ -0,0 +1,272 @@
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeyCommand
+  name: command encryption key
+  description: An encryption key used by crew's bosses.
+  components:
+  - type: EncryptionKey
+    channels:
+    - Common
+    - Command
+    - Engineering
+    - Medical
+    - Science
+    - Security
+    - Service
+    - Supply
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: com_cypherkey
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeyEngineering
+  name: engineering encryption key
+  description: An encryption key used by the engineers.
+  components:
+  - type: EncryptionKey
+    channels:
+    - Common
+    - Engineering
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: eng_cypherkey
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeyCE
+  name: chief engineer's encryption key
+  description: An encryption key used by the boss of the real men.
+  components:
+  - type: EncryptionKey
+    channels:
+    - Common
+    - Command
+    - Engineering
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: ce_cypherkey
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeyMedical
+  name: medical encryption key
+  description: An encryption key used by those who save lives.
+  components:
+  - type: EncryptionKey
+    channels:
+    - Common
+    - Medical
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: med_cypherkey
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeyCMO
+  name: chief medical officer encryption key
+  description: An encryption key used by the chief of doctors.
+  components:
+  - type: EncryptionKey
+    channels:
+    - Common
+    - Command
+    - Medical
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: cmo_cypherkey
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeyMedicalScience
+  name: med-sci encryption key
+  description: An encryption key used by someone who hasn't decided which side to take.
+  components:
+  - type: EncryptionKey
+    channels:
+    - Common
+    - Medical
+    - Science
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: medsci_cypherkey
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeyScience
+  name: science encryption key
+  description: An encryption key used by scientists. Maybe it is plasmaproof?
+  components:
+  - type: EncryptionKey
+    channels:
+    - Common
+    - Science
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: sci_cypherkey
+
+- type: entity
+  parent: EncryptionKey
+  id: EncryptionKeyRobo
+  name: robotech encryption key
+  description: An encryption key used by robototech engineers. Maybe it has a LAH-6000 on it?
+  components:
+  - type: EncryptionKey
+    channels:
+    - Common
+    - Science
+  - type: Sprite
+    sprite: Objects/Devices/encryption_keys.rsi
+    state: rob_cypherkey
```

Comment:
What's this? Robotics didn't have its own channel on any of the SS13 servers I played on, and we don't even have a roboticist role yet.

Reply:
Some servers made robotist sci-engineering job, so it will be useful, for example baystation port